### PR TITLE
express example

### PR DESCRIPTION
**Pull request by ZYJLiu**
This example demonstrates a Solana Action built with Node.js using the Express
framework. It provides an API endpoint for transferring SOL.

## Setup

Navigate to `./examples/express`.

Install dependencies:

```
npm install
```

Start the application:

```
npm start
```

The server will start running on http://localhost:8080.

The endpoint for the Action is: http://localhost:8080/api/actions/transfer-sol

You can test the Action on devnet as a Blink at https://dial.to/devnet:
https://www.dial.to/devnet?action=solana-action%3Ahttp%3A%2F%2Flocalhost%3A8080%2Fapi%2Factions%2Ftransfer-sol

### CODE REVIEW

File: examples/express/server.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        toPubkey,
+        lamports: amount * LAMPORTS_PER_SOL,
+      })
+    );
+    transaction.feePayer = fromPubkey;
+
+    const { blockhash } = await connection.getLatestBlockhash();
+    transaction.recentBlockhash = blockhash;
+
+    const serializedTransaction = transaction.serialize({ requireAllSignatures: false });
```
Comment (nickfrosty) [inline examples/express/server.js:104]: Can you update this to use the `createPostResponse` helper function:
https://solana-developers.github.io/solana-actions/functions/createPostResponse.html

**Comment by nickfrosty** [comment]
This is awesome and was on the todo list! Glad you were able to get to it :)
One nit: see my review comment

File: examples/express/server.js (hunk 1 above)
Comment (ZYJLiu) [reply examples/express/server.js:104]: I'm getting the following error when I try to start the express server after installing and importing `@solana/actions`.

```
> express@1.0.0 start
> node server

node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/johnliu/code/solana-actions-fork/examples/express/node_modules/@solana/actions/lib/esm/constants' imported from /Users/johnliu/code/solana-actions-fork/examples/express/node_modules/@solana/actions/lib/esm/createPostResponse.js
    at new NodeError (node:internal/errors:405:5)
    at finalizeResolution (node:internal/modules/esm/resolve:327:11)
    at moduleResolve (node:internal/modules/esm/resolve:946:10)
    at defaultResolve (node:internal/modules/esm/resolve:1132:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:835:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v18.18.0
```

Installed with:
```
npm i @solana/actions  
```

Package.json dependencies look like this:
```
  "dependencies": {
    "@solana/actions": "^1.0.3",
    "@solana/web3.js": "^1.94",
    "cors": "^2.8.5",
    "express": "^4.19.2"
  }
 ```
 
 Importing like this in server.js:
 ```
 import {
  createPostResponse,
} from "@solana/actions";
```

Any idea how to fix?

Comment (ZYJLiu) [reply examples/express/server.js]: works with `"@solana/actions": "^1.1.1"`

**Pull request merged**